.gitignore

```diff
@@ -18,3 +18,4 @@
 /ui/.next/
```

Comment:
Can you add this comment to the top of the file?

Since these ignore rules are shared between Git and Docker, every rule should take one of three forms:
**/SOME_PATTERN
/SOME_PATTERN
!/SOME_PATTERN